### ISSUE TITLE
Pinned tabs are doubled

### ISSUE THREAD

**Issue by nishi2x4** [yfdyh000/tabutils#158]
![tabutils](https://user-images.githubusercontent.com/22878838/29748411-6cf8e210-8b51-11e7-9958-aa3122c90dea.gif)

Windows 7 64bit
Firefox 55.0.2
Tab Utilities Fixed 1.5.2017.08.17
Profile is new, and Add-ons other than TabUtilsFixed are not installed

**Comment by yfdyh000** [comment]
I see different unusual behavior in Firefox 55.0.3 with TUF 1.5.2017.08.17, blank space after pin tab.

Try set the `extensions.tabutils.pinTab.showPhantom` to false in about:config if you would.

**Comment by nishi2x4** [comment]
When setting that option, pin tab got the correct behavior.
thanks!

**Comment by LL25255252** [comment]
Yes, setting extensions.tabutils.pinTab.showPhantom to false in about:config fixes this pinned tab issue.
So, @yfdyh000 , Can you please fix it and upload another last version. If not ,the addon will start getting many one star reviews...
Thanks in advance.

**Comment by yfdyh000** [comment]
@LL25255252 I don't know why it happened for now, and I also don't want to troubleshoot the source of this problem, as this sometimes consumes for a number of hours...
